modules/persistent-storage-csi-vol-detach-non-graceful-shutdown-procedure.adoc

```diff
@@ -32,6 +32,11 @@ If the node is not completely shut down, do not proceed with tainting the node.
 +
 . Taint the corresponding node object by running the following command:
 +
+[IMPORTANT]
+====
+Tainting a node this way deletes all pods on that node.
+====
++
```

Comment:
@gcharot may be we can improve this wording to say. 

```
Tainting a node this way deletes all pods on that node. This will also cause any pods that are backed by statefulsets to be evicted and replacement pods to be created on different node
```

Reply:
Fixed.